Makefile

```diff
@@ -70,6 +70,10 @@ release-static-armv7:
 	mkdir -p build/release
 	cd build/release && cmake -D BUILD_TESTS=OFF -D ARCH="armv7-a" -D STATIC=ON -D BUILD_64=OFF -D CMAKE_BUILD_TYPE=release ../.. && $(MAKE)
 
+release-static-android:
```

Comment:
TBH, i don't understand PIC/PIE.
> only` position independent executables (PIE) are supported

is the message i got when i tried to execute it  whitout PIE
this [doc page](https://developer.android.com/ndk/guides/cmake.html) and [this SE](http://stackoverflow.com/questions/30498776/position-independent-executables-and-android-lollipop/) made me add it
this seems to only be mandatory from android 5 and won't work with android < 4.1, so we may want to add a variable for the android api level and check it, but i don't think it's worth it to support prior versions